docs-content/getting-started/fullstack-frameworks/next-js/5_api-app-router.md

```diff
@@ -1,66 +1,81 @@
 ---
-title: Next.js Api (App Router)
+title: App Router Api
 slug: app-router
-heading: Next.js Api (App Router)
+heading: Next.js App Router Api
 createdAt: 2023-10-03T00:00:00.000Z
 updatedAt: 2023-10-03T00:00:00.000Z
 ---
 
 ## Installation
 
 ```shell
-# with npm
 npm install @highlight-run/next
 ```
 
 ## API Route Instrumentation
 
+Node.js 
+
 ```hint
-This section applies to Next.js App Router routes only. Each App Router route must be wrapped individually.
+ Each App Router route must be wrapped individually.
 ```
 
 ######
 
-1. Create a file to export your `AppRouterHighlight` wrapper function:
+1. Add `@highlight-run/node` to `experimental.serverComponentsExternalPackages` in your `next.config.js`. 
+
+```javascript
+// next.config.js
+const nextConfig = {
```

Comment:
could we create some reminder ticket to come back to this once https://github.com/vercel/next.js/pull/57783 is merged to remove this since it wont be needed?